code/obj/machinery/computer/cloning.dm

```diff
@@ -447,6 +447,11 @@ proc/eligible_to_clone(var/datum/mind/mind)
 	if (!M.client)
 		return null
 
+	if (istype(M, /mob/new_player))
+		var/mob/new_player/player = M
+		if (player.is_respawned_player)
+			return null
```

Comment:
Actually is there ever a situation where we want a `new_player` to be cloned anyway?